gcapi/gcapi.py

```diff
@@ -154,19 +158,15 @@ def detail(self, pk):
 class ModifiableMixin:
     _client = None  # type: Client
 
-    modify_json_schema = None  # type: jsonschema.Draft7Validator
-
     def __init__(self):
         pass
 
-    def _process_post_arguments(self, post_args):
-        if self.modify_json_schema is not None:
-            self.modify_json_schema.validate(post_args)
-
-    def _validate_data(self, data):
+    def _process_request_arguments(self, method, data):
         if data is None:
             data = {}
-        self._process_post_arguments(data)
+        schema = self.validation_schemas.get(method)
+        if schema:
```

Comment:
`if schema is None`? What if the schema is `{}`, is this possible?

Reply:
Like I wrote above. It is either not in the dict or a Draft7Validator instances. It is never a bare json dict. I think I even added the type annotation!